src/applications/facility-locator/components/search-results/LocationInfoBlock.jsx

```diff
@@ -25,9 +26,15 @@ const LocationInfoBlock = ({ location }) => {
         </span>
       ) : (
         <span>
-          <Link to={`facility/${location.id}`}>
-            <h2 className="vads-u-font-size--h5">{name}</h2>
-          </Link>
+          {isVADomain(website) && environments.isStaging() ? (
```

Comment:
what use case is not covered by the environment flag?

Reply:
We just want this code to run on staging for QA

Reply:
@rianfowler can you provide more feedback otherwise approve it?